commander/src/utils/reader.ts

```diff
@@ -352,3 +357,35 @@ export const getParamsFromPrompt = async (
 		? transformNestedAsset(assetSchema, filteredResult)
 		: transformAsset(assetSchema, result as Record<string, string>);
 };
+
+export const getFileExtension = (filePath: string): string => {
+	const ext = path.extname(filePath);
+
+	if (!ext || ext !== '.json') {
+		throw new ValidationError(INVALID_JSON_FILE);
+	}
+
+	return ext;
+};
+
+export const readParamsFile = (filePath: string): string => {
+	try {
+		const params = fs.readFileSync(filePath, 'utf8');
+		return params;
+	} catch (err) {
+		throw new ValidationError(FILE_NOT_FOUND);
+	}
+};
+
+export const getFileParams = (filePath: string): string => {
+	let data = '';
+	const ext = getFileExtension(filePath);
+
+	if (ext === '.json') {
+		data = readParamsFile(filePath);
+	} else {
+		throw new ValidationError(INVALID_JSON_FILE);
+	}
```

Comment:
I think this case is already checked in function `getFileExtension` at L382 and probably redundant.

Reply:
This is correct. The check is redundant.

Reply:
Updated!